scripts/pylib/twister/twisterlib.py

```diff
@@ -58,6 +57,14 @@
 except ImportError:
     print("Install psutil python module with pip to run in Qemu.")
 
+try:
+    import pty
+except ImportError as capture_error:
+    if os.name == "nt":  # "nt" means that program is running on Windows OS
+        pass  # "--device-serial-pty" option is not supported on Windows OS
+    else:
+        raise capture_error
```

Comment:
```suggestion
if os.name != "nt":
    import pty
```

Reply:
I agree, that your proposition is shorter and correct, but I recommend more Pythonic way with `try-except` statement (read more about EAFP on Python documentation: https://docs.python.org/3/glossary.html#term-EAFP). I also chose this solution due to the fact that in lines 37-38 are placed similar `try-except` statements. But if it your suggestion is very important to you, please explain it to me and I'll introduce it :)